src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,16 @@
 package core.basesyntax;
 
 public class MainApp {
+    public static void main(String[] args) {
 
+        Machine[] machines = new Machine[3];
+        machines[0] = new Truck();
+        machines[1] = new Bulldozer();
+        machines[2] = new Excavator();
+
+        for (Machine i : machines) {
```

Comment:
Consider using a more descriptive variable name instead of 'i' for the loop variable, such as 'machine', to improve code readability.